frontend/src/store/index.js

```diff
@@ -1207,14 +1213,18 @@ const actions = {
     const { data } = await getKubeconfigData()
     commit('SET_KUBECONFIG_DATA', data)
   },
-  async ensureProjectTerminalShortcutsLoaded ({ commit, state }) {
+  async ensureProjectTerminalShortcutsLoaded ({ commit, dispatch, state }) {
     const { namespace, projectTerminalShortcuts } = state
     if (!projectTerminalShortcuts || projectTerminalShortcuts.namespace !== namespace) {
-      const { data: items } = await listProjectTerminalShortcuts({ namespace })
-      commit('SET_PROJECT_TERMINAL_SHORTCUTS', {
-        namespace,
-        items
-      })
+      try {
+        const { data: items } = await listProjectTerminalShortcuts({ namespace })
+        commit('SET_PROJECT_TERMINAL_SHORTCUTS', {
+          namespace,
+          items
+        })
+      } catch (err) {
+        dispatch('setError', err)
```

Comment:
In case the backend responds with an error, this message is shown on the cluster details page without any context.  On the cluster list, no error is shown even though it runs into this error.

<img width="421" alt="Screenshot 2020-09-16 at 21 24 06" src="https://user-images.githubusercontent.com/5526658/93382816-fccd0700-f862-11ea-8fcd-0a7019145a14.png">

Alternatively, just log the error to the console and add a bit more context, like `Failed to list project terminal shortcuts`

Reply:
What do you propose:
1. Log the error to the console as `Failed to list project terminal shortcuts`
2. Show an error to the user that says `Failed to list project terminal shortcuts`
3. Ignore the error completely.

Reply:
I do not have a strong opinion here. I guess I would vote for 1 in this case